lib/ReactGridLayout.jsx

```diff
@@ -699,6 +702,14 @@ export default class ReactGridLayout extends React.Component<Props, State> {
   }
 
   onDragOver = (e: DragOverEvent) => {
+    // ignore events from layout's children
+    if (
```

Comment:
In my testing, simply adding `event.dataTransfer.setData('text/plain', '')` fixes the bug. Why is this exception needed?

Reply:
in case of firefox dragging fires events with different targets and it makes dragging inaccurate (the placeholder can jump unpredictably without this code)

Reply:
Ah I can reproduce that. Thanks. Could you update the comment to indicate this for the future?

Reply:
Ah I can reproduce that. Thanks. Could you update the comment to indicate this for the future?

Reply:
Ah I can reproduce that. Thanks. Could you update the comment to indicate this for the future?